include/xrpl/protocol/LedgerFormats.h

```diff
@@ -52,150 +52,13 @@ namespace ripple {
-
-        \sa keylet::signers
-     */
-    ltSIGNER_LIST = 0x0053,
-
-    /** A ledger object which describes an offer on the DEX.
-
-        \sa keylet::offer
-     */
-    ltOFFER = 0x006f,
-
-
-    /** The ledger object which lists details about sidechains.
-
-        \sa keylet::bridge
-    */
-    ltBRIDGE = 0x0069,
-
-    /** A ledger object that contains a list of ledger hashes.
-
-        This type is used to store the ledger hashes which the protocol uses
-        to implement skip lists that allow for efficient backwards (and, in
-        theory, forward) forward iteration across large ledger ranges.
-
-        \sa keylet::skip
-     */
-    ltLEDGER_HASHES = 0x0068,
-
-    /** The ledger object which lists details about amendments on the network.
-
-        \note This is a singleton: only one such object exists in the ledger.
-
-        \sa keylet::amendments
-     */
-    ltAMENDMENTS = 0x0066,
-
-    /** A claim id for a cross chain transaction.
-
-        \sa keylet::xChainClaimID
-    */
-    ltXCHAIN_OWNED_CLAIM_ID = 0x0071,
-
-    /** A claim id for a cross chain create account transaction.
-
-        \sa keylet::xChainCreateAccountClaimID
-    */
-    ltXCHAIN_OWNED_CREATE_ACCOUNT_CLAIM_ID = 0x0074,
-
-    /** The ledger object which lists the network's fee settings.
-
-        \note This is a singleton: only one such object exists in the ledger.
-
-        \sa keylet::fees
-     */
-    ltFEE_SETTINGS = 0x0073,
-
-    /** A ledger object describing a single escrow.
-
-        \sa keylet::escrow
-     */
-    ltESCROW = 0x0075,
-
-    /** A ledger object describing a single unidirectional XRP payment channel.
-
-        \sa keylet::payChan
-     */
-    ltPAYCHAN = 0x0078,
-
-    /** A ledger object which describes a check.
-
-        \sa keylet::check
-     */
-    ltCHECK = 0x0043,
-
-    /** A ledger object which describes a deposit preauthorization.
-
-        \sa keylet::depositPreauth
-     */
-    ltDEPOSIT_PREAUTH = 0x0070,
-
-    /** The ledger object which tracks the current negative UNL state.
-
-        \note This is a singleton: only one such object exists in the ledger.
-
-        \sa keylet::negativeUNL
-     */
-    ltNEGATIVE_UNL = 0x004e,
-
-    /** A ledger object which contains a list of NFTs
-
-        \sa keylet::nftpage_min, keylet::nftpage_max, keylet::nftpage
-     */
-    ltNFTOKEN_PAGE = 0x0050,
-
-    /** A ledger object which identifies an offer to buy or sell an NFT.
-
-        \sa keylet::nftoffer
-     */
-    ltNFTOKEN_OFFER = 0x0037,
-
-    /** The ledger object which tracks the AMM.
-
-       \sa keylet::amm
-    */
-    ltAMM = 0x0079,
-
-    /** The ledger object which tracks the DID.
-
-       \sa keylet::did
-    */
-    ltDID = 0x0049,
-
-    /** A ledger object which tracks Oracle
-        \sa keylet::oracle
-     */
-    ltORACLE = 0x0080,
+#pragma push_macro("OBJECT")
+#undef OBJECT
+#define OBJECT(tag, value, name, fields) tag = value,
```

Comment:
For consistency, please follow the same style as in `Feature.h` (vertical spacing)